src/main/com/smithsmodding/armory/common/logic/initialization/CommonSystemInitializer.java

```diff
@@ -49,87 +45,27 @@ public static CommonSystemInitializer getInstance() {
-        GameRegistry.registerTileEntity(TileEntityConduit.class, References.InternalNames.TileEntities.Conduit);
-        GameRegistry.registerTileEntity(TileEntityMoltenMetalTank.class, References.InternalNames.TileEntities.Tank);
-        GameRegistry.registerTileEntity(TileEntityPump.class, References.InternalNames.TileEntities.Pump);
-        GameRegistry.registerTileEntity(TileEntityMoltenMetalMixer.class, References.InternalNames.TileEntities.MoltenMetalMixer);
-    }
-
-    private static void registerCreativeTabs() {
-        ModCreativeTabs.GENERAL = new GeneralTabs();
-        ModCreativeTabs.COMPONENTS = new ComponentsTab();
-        ModCreativeTabs.HEATEDITEM = new HeatedItemTab();
-        ModCreativeTabs.ARMOR = new ArmorTab();
-
-        ModItems.IT_CHAIN.setCreativeTab(ModCreativeTabs.GENERAL);
-        ModItems.IT_GUIDE.setCreativeTab(ModCreativeTabs.GENERAL);
-        ModItems.IT_HAMMER.setCreativeTab(ModCreativeTabs.GENERAL);
-        ModItems.IT_INGOT.setCreativeTab(ModCreativeTabs.GENERAL);
-        ModItems.IT_NUGGET.setCreativeTab(ModCreativeTabs.GENERAL);
-        ModItems.IT_PLATE.setCreativeTab(ModCreativeTabs.GENERAL);
-        ModItems.IT_RING.setCreativeTab(ModCreativeTabs.GENERAL);
-        ModItems.IT_TONGS.setCreativeTab(ModCreativeTabs.GENERAL);
-
-        ModItems.IT_COMPONENT.setCreativeTab(ModCreativeTabs.COMPONENTS);
-
-        ModItems.IT_HEATEDITEM.setCreativeTab(ModCreativeTabs.HEATEDITEM);
-
-        ModItems.Armor.IT_CHESTPLATE.setCreativeTab(ModCreativeTabs.ARMOR);
-        ModItems.Armor.IT_HELMET.setCreativeTab(ModCreativeTabs.ARMOR);
-        ModItems.Armor.IT_LEGGINGS.setCreativeTab(ModCreativeTabs.ARMOR);
-        ModItems.Armor.IT_SHOES.setCreativeTab(ModCreativeTabs.ARMOR);
-
-        ModBlocks.BL_ANVIL.setCreativeTab(ModCreativeTabs.GENERAL);
-        ModBlocks.BL_CONDUIT.setCreativeTab(ModCreativeTabs.GENERAL);
-        ModBlocks.BL_FIREPLACE.setCreativeTab(ModCreativeTabs.GENERAL);
-        ModBlocks.BL_FORGE.setCreativeTab(ModCreativeTabs.GENERAL);
-        ModBlocks.BL_PUMP.setCreativeTab(ModCreativeTabs.GENERAL);
-        ModBlocks.BL_TANK.setCreativeTab(ModCreativeTabs.GENERAL);
-        ModBlocks.BL_RESOURCE.setCreativeTab(ModCreativeTabs.GENERAL);
-    }
-
-    private static void removeRecipes() {
+    private static void removeRecipes()
+    {
         if (!ArmoryConfig.enableHardModeNuggetRemoval)
+        {
             return;
+        }
 
-        ListIterator<IRecipe> iterator = CraftingManager.getInstance().getRecipeList().listIterator();
-        while (iterator.hasNext()) {
+        Iterator<IRecipe> iterator = CraftingManager.REGISTRY.iterator();
+        while (iterator.hasNext())
+        {
             IRecipe recipe = iterator.next();
             tryRemoveRecipeFromGame(recipe, iterator);
         }
     }
 
+    @Override
+    public void onPostInit(@Nonnull FMLPostInitializationEvent event) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)